### PR TITLE
improve lang JA

### PR DESCRIPTION
**Pull request by 44uk**

### CODE REVIEW

File: src/config/i18n/ja.json [hunk 1]
```diff
@@ -377,30 +377,30 @@
...
-	"nodeStatsTitle": "Node Statistics",
-	"allNodes": "Total count",
-	"nodeCountByRoles": "Count By Roles",
-	"chainInfo": "Chain Info",
+	"previousImportanceBlockHash": "前インポータンスブロックハッシュ",
+	"nodeStatsTitle": "ノード情報",
+	"allNodes": "合計数",
+	"nodeCountByRoles": "ロールごとの数",
+	"chainInfo": "チェーン情報",
 	"nodeHeightStatsTitle": "Count of nodes by chain height",
```
Comment (AnthonyLaw) [inline src/config/i18n/ja.json:385]: can you delete those space

Comment (44uk) [reply src/config/i18n/ja.json]: replace tab to space indent.

Comment (AnthonyLaw) [reply src/config/i18n/ja.json]: thank you mate 👍🏼

**Review by AnthonyLaw — APPROVED** [review]
thank you very much.

**Pull request merged**